README.md

```diff
@@ -6,6 +6,18 @@ instances, attribute hashes, and stubbed objects), and support for multiple
 factories for the same class (`user`, `admin_user`, and so on), including factory
 inheritance.
 
+### Transitioning from factory\_girl\_rails?
+
+Transitioning is a two step process:
+
+1) Replace `factory_girl_rails` with `factory_bot_rails` in your Gemfile
```

Comment:
Possible to use Markdown for these bullet-points?

Reply:
I think this is Markdown? Github's preview for this file seems to be parsing it as valid Markdown

Reply:
TIL! That's pretty cool, I think this is easier to read.